bigtable/tests/mutations_integration_test.cc

```diff
@@ -255,17 +256,18 @@ TEST_F(MutationIntegrationTest, DeleteFromColumnForReversedTimestampRangeTest) {
   };
 
   CreateCells(*table, created_cells);
+
 #if GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
   // Try to delete the columns with an invalid range:
-  EXPECT_THROW(
-      table->Apply(bigtable::SingleRowMutation(
-          key, bigtable::DeleteFromColumn(column_family2, "c2", 4000, 2000))),
-      bigtable::PermanentMutationFailure);
+  EXPECT_THROW(table->Apply(bigtable::SingleRowMutation(
+                   key, bigtable::DeleteFromColumn(column_family2, "c2",
+                                                   4000_ms, 2000_ms))),
```

Comment:
Why not use `4_ms` and `2_ms` here?

Reply:
Yes like your earlier comment, due to my  wrong understanding about default raw integer is millisecond, I did mistake here as well.
so here 4000_us and 2000_us is more obvious, and also I have to remove all the values multiplied by 1000, to make code less change and more readable.
Thanks for comments and suggestion.